addons/l10n_it_stock_ddt/tests/test_edi.py

```diff
@@ -126,12 +126,11 @@ def test_deferred_invoice(self):
             deferred_invoice.action_post()
 
         # Check the XML output of the invoice
-        self._test_invoice_with_sample_file(
-            deferred_invoice,
-            "deferred_invoice.xml",
-            xpaths_result={"//DatiGeneraliDocumento/Numero": "<Numero/>",},
-            xpaths_file={"//DatiGeneraliDocumento/Numero": "<Numero/>",}
-        )
+        invoice_xml = self.edi_format._l10n_it_edi_export_invoice_as_xml(deferred_invoice)
```

Comment:
What are the reasons for these changes in l10n_it_stock_ddt (maybe separate commit?)

Reply:
I'm sorry, it cannot be done.
These changes are there because I moved some data from a test class to its base class, and this inherits, so the two modifications are linked and necessary together. The old function `_test_invoice_with_sample_file` reads from the wrong folder.